docs/templates/backend.md

```diff
@@ -45,6 +45,22 @@ KERAS_BACKEND=tensorflow python -c "from keras import backend"
 Using TensorFlow backend.
 ```
 
+In keras it is possible to load more backends than `"tensorflow"`, `"theano"`, and `"cntk"`. keras can use external backends as well, and this can be performed by changing the `keras.json` configuration file, and the `"backend"` setting. Suppose you have a python file called mxnet_backend.py that you wanted to use as your external backend. The `keras.json` configuration file would be changed as follows:
```

Comment:
MXNet is not a good example, because there is, in fact, a Keras MXNet which is a fork and cannot be used as Keras backend.